codice/ddf
# DDF-6012 Move Admin-UI to a new repo

#### What does this PR do?
- Experiments with the idea of moving out the DDF Admin UI
- Removes the UI plugin, it is currently in this branch: https://github.com/codice/ddf-admin-ui
- Removes Frontend Maven Plugin and a few UI related properties

#### Who is reviewing it? 

@blen-desta 
@TonyMorrison 

#### Select relevant component teams: 
@codice/build 
@codice/continuous-integration 
@codice/security 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@pklinef 
@andrewkfiedler 

#### How should this be tested?
I currently updated the admin-ui version to prove out DDF pulling in the new version. I was able to verify install, login, and using the admin UI with the new version.

#### Any background context you want to provide?
Frontend-maven-plugin showed 12% of the build in maven build scanner

#### What are the relevant tickets?
Fixes: #6012 

#### Screenshots
![Screen Shot 2020-04-28 at 4 48 00 PM](https://user-images.githubusercontent.com/4359166/80548592-20ed8180-8970-11ea-9f0b-3878ffe60ebb.png)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/codice/ddf/6019)
